packages/front-end/styles/_bootstrap-theme-overrides.scss

```diff
@@ -63,6 +63,11 @@ pre {
 
   [class$="-Input"] {
     color: var(--text-color-main);
+
+    // to avoid issues with react-select adding `color: inherit`
+    input {
+      color: var(--text-color-main) !important;
+    }
```

Comment:
This is an attempt to fix the illegible text colour in an input field that can happen occasionally. There's a good chance this doesn't have any effect so if we experience issues with this, we should delete it.